metron-interface/metron-alerts/src/index.html

```diff
@@ -16,7 +16,7 @@
 <head>
   <meta charset="utf-8">
   <title>Metron Alerts</title>
-  <base href="/">
+  <base href="./">
```

Comment:
What was the reason for changing the base href to a relative path? It looks like this is breaking the Cypress tests running on Travis.

Reply:
Disregard my previous comment. This is not the cause of the failing Cypress tests.